src/platform/packages/shared/kbn-esql-ast/src/definitions/utils/autocomplete/functions.ts

```diff
@@ -358,15 +360,18 @@ export async function getFunctionArgsSuggestions(
         location = Location.STATS_TIMESERIES;
       }
       suggestions.push(
-        ...getFunctionSuggestions({
-          location,
-          returnTypes: canBeBooleanCondition
-            ? ['any']
-            : (ensureKeywordAndText(
-                getTypesFromParamDefs(typesToSuggestNext)
-              ) as FunctionParameterType[]),
-          ignored: fnToIgnore,
-        }).map((suggestion) => ({
+        ...getFunctionSuggestions(
+          {
+            location,
+            returnTypes: canBeBooleanCondition
+              ? ['any']
+              : (ensureKeywordAndText(
+                  getTypesFromParamDefs(typesToSuggestNext)
+                ) as FunctionParameterType[]),
+            ignored: fnToIgnore,
+          },
+          hasMinimumLicenseRequired
```

Comment:
P.S. Not for this PR, just comment on the codebase. Adding a new callback now requires us to pass through `hasMinimumLicenseRequired` multiple functions deep and makes code less readable/maintainable every time.

We really should introduce a suggestion context, which contains all information needed for a single suggestion, like [suggested here](https://github.com/elastic/kibana-team/discussions/967#discussioncomment-13322076).